### PR TITLE
Refactor mqtt handler

### PR DESCRIPTION
**Pull request by viktorsmari**
- Return early if topic is nil
- Return early if device is nil
- Remove Raven capture if device was nil - was flooding our Sentry
- Add test for empty inventory topic

### CODE REVIEW

File: app/lib/mqtt_messages_handler.rb [hunk 1]
```diff
@@ -30,20 +33,12 @@ def self.handle_readings(topic, message)
...
-    device_token = self.device_token(topic)
-
-    return if device_token.blank?
-
-    device = Device.find_by(device_token: device_token)
-    if device.present?
-      payload[:device_id] = device.id
-    end
+    payload[:device_id] = device.id
+    payload[:device_token] = device.device_token # TODO: remove after migration
```
Comment (viktorsmari) [inline app/lib/mqtt_messages_handler.rb:39]: @pral2a do you remember if we should remove this line now?

Comment (pral2a) [reply app/lib/mqtt_messages_handler.rb:39]: Yes, we can remove it, finally! That is an important decision as it fixes a critical vulnerability
